pyanaconda/network.py

```diff
@@ -94,10 +94,11 @@ def check_ip_address(address, version=None):
         return False
 
 
-def is_valid_hostname(hostname):
+def is_valid_hostname(hostname, local=False):
     """Check if the given string is (syntactically) a valid hostname.
 
-    :param hostname: a string to check
+    :param str hostname: a string to check
+    :param bool local: is the hostname static (for this system) or not (anywhere on the network)
```

Comment:
I would remove `anywhere` here, `on the network` seems good enough to me.

Reply:
Done!